.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Can we omit this file please?

Reply:
Oh, there's a bunch of files proposed to be added that should be ignored. Please remove all those

Reply:
yes, will do!